### PR TITLE
[DOM-140] Response 200 status code with a message id in send message endpoint

### PR DESCRIPTION
**Pull request by ms-darianbenito**
After make all send message steps asynchronous We will to response a `202-accepted` status code instead.

- [x] Tinking on tests

Related to [DOM-140](https://makingsense.atlassian.net/browse/DOM-140)

### CODE REVIEW

File: Doppler.PushContact/Controllers/PushContactController.cs [hunk 1]
```diff
@@ -108,9 +108,12 @@ public async Task<IActionResult> Message([FromRoute] string domain, [FromBody] M
                 await _pushContactService.AddHistoryEventsAsync(pushContactHistoryEvents);
             }
 
-            // TODO: response a MessageId and run all steps asynchronous
+            // TODO: run all steps asynchronous
+            // and response an 202-accepted with the message id instead
 
-            throw new NotImplementedException();
+            var messageId = Guid.NewGuid();
```
Comment (andresmoschini) [inline Doppler.PushContact/Controllers/PushContactController.cs:114]: What do you think about moving this line at the beginning and pass the id to `_messageSender.SendAsync`?

Comment (ms-darianbenito) [reply Doppler.PushContact/Controllers/PushContactController.cs]: *Perdón Andres, estaba concentrado haciendo unos cambios en este PR, por eso tu comentario figura como `outdated`. En cuanto pueda analizo tu sugerencia

Comment (ms-darianbenito) [reply Doppler.PushContact/Controllers/PushContactController.cs]: > What do you think about moving this line at the beginning and pass the id to _messageSender.SendAsync?

`_messageSender.SendAsync` just sends the message across `PushApi`, but what do you think about creating an `IMessageWorker` with a `HandleMessageAsync(Guid messageId, ...)` method with all logic inside? does this make sense thinking about our end goal?
image

Something like this:
image

Comment (ms-darianbenito) [reply Doppler.PushContact/Controllers/PushContactController.cs]: A modo de ilustrar como quedaría nuestro esquema parcial, hasta realizar la tarea de la asincronía, luego del merge de este PR:
image

**Pull request merged**